frontend/templates/base.html

```
@@ -1,4 +1,4 @@
-{% load webpack_loader static %}
+{% load webpack_loader static seo_tags %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)